guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -0,0 +1,356 @@
+
+![Flow Builder create tag](../../../.gitbook/assets/flow-builder-tag.png)
+
+```tsx
+import template from './swag-example-plugin-modal.html.twig';
+const { Component } = Shopware;
+
+Component.register('swag-example-plugin-modal', {
+    template,
+
+    props: {
+        sequence: {
+            type: Object,
+            required: true,
+        },
+    },
+
+    data() {
+        return {
+            tags: [],
+        };
+    },
+
+    created() {
+        this.createdComponent();
+    },
+
+    methods: {
+        createdComponent() {
+            this.tags = this.sequence?.config?.tags || [];
+        },
+
+        onClose() {
+            this.$emit('modal-close');
+        },
+
+        onAddAction() {
+            const sequence = {
+                ...this.sequence,
+                config: {
+                    ...this.config,
+                    tags: this.tags
+                },
+            };
+
+            this.$emit('process-finish', sequence);
+        },
+    },
+});
+```
+
+```html
+{% block swag_example_plugin_modal %}
+    <sw-modal
+        class="swag-example-plugin-modal"
+        :title="$tc('swag-example-plugin.titleCreateTag')"
+        @modal-close="onClose"
+    >
+        {% block swag_example_plugin_modal_content %}
+            <sw-multi-tag-select
+                v-model="tags"
+                :label="$tc('swag-example-plugin.labelTags')"
+            />
+        {% endblock %}
+
+        {% block swag_example_plugin_modal_footer %}
+            <template #modal-footer>
+                {% block swag_example_plugin_modal_footer_cancel_button %}
+                    <sw-button
+                        class="swag-example-plugin-modal__cancel-button"
+                        size="small"
+                        @click="onClose"
+                    >
+                        {{ $tc('global.default.cancel') }}
+                    </sw-button>
+                {% endblock %}
+
+                {% block swag_example_plugin_modal_footer_save_button %}
+                    <sw-button
+                        class="swag-example-plugin-modal__save-button"
+                        variant="primary"
+                        size="small"
+                        @click="onAddAction"
+                    >
+                        {{ sequence.id
+                        ? $tc('swag-example-plugin.buttonSaveAction')
+                        : $tc('swag-example-plugin.buttonAddAction') }}
+                    </sw-button>
+                {% endblock %}
+            </template>
+        {% endblock %}
+    </sw-modal>
+{% endblock %}
+```
+
+### Demo
+Now you can view the demo for this custom Flow builder triger and action as below.
+
+![Flow Builder demo](../../../.gitbook/assets/flow-builder-demo.gif)
+
+
+The source code for this demo you could find it at [Flow Builder ExamplePlugin](https://github.com/huytdq94/sw-example-plugin-flow-builder).
+
+<div data-gb-custom-block data-tag="hint" data-style='info'>
```

Comment:
This is not going to display properly. Please use gitbook syntax and add the hint at the beginning of the article.

For reference:
 * https://gitbook.gitbook.io/git-sync/hints
 * https://github.com/GitbookIO/git-sync-normalization/blob/master/hints.md

The correct markup for displaying hints is this:

{% hint style="info" %}
This functionality is available starting with Shopware 6.4.3.0.
{% endhint %}